inductiva/mixins/file_manager.py

```diff
@@ -0,0 +1,208 @@
+"""Mixin for file management.
+
+Methods to manage files and directories, within your local computer in
+order to prepare them for the simulation process. This mixin can be
+used to copy files and directories, and render template files with
+the given set of arguments.
+"""
+import glob
+import os
+import shutil
+import pathlib
+import re
+
+from absl import logging
+import jinja2
+
+from inductiva.utils import files
+
+TEMPLATE_EXTENSION = ".jinja"
+
+
+class FileManager:
+    """Class for file management."""
+    DEFAULT_ROOT_DIR = "input_dir"
+    __root_dir = None  #pylint: disable=invalid-name
+
+    def __check_root_dir(self):
+        """Check if the root directory is set, and set it otherwise."""
+        if self.__root_dir is None:
+            self.set_root_dir(self.DEFAULT_ROOT_DIR)
+
+    def set_root_dir(self, root_dir=None):
+        """Set a root directory for the file manager.
+        
+        Args:
+            root_dir: Path to the root directory.
+                If None, an error is raised.
+                If the directory already exists. A new directory
+                name is created following a sequence of #1, #2, #3, etc."""
+
+        if root_dir is None:
+            raise ValueError("Given root directory cannot be None")
+        elif os.path.isdir(root_dir):
+            generated_root_dir = gen_name(root_dir)
```

Comment:
```suggestion
        elif os.path.isdir(root_dir):
            if os.getenv('INDUCTIVA_DISABLE_FILEMANAGER_AUTOSUFFIX', False):
                raise FileExistsError(f'Directory {root_dir} already exists.')
            generated_root_dir = gen_name(root_dir)
```

Reply:
Shall this be set with Inductiva package, like, `inductiva.filemanager_autosuffix = True`, as the other env variables we have?

Reply:
Wondering about this further... Shouldn't we allow override as well? I don't see the point of disable just to raise an error. Could you elaborate on this?